src/lib/components/ContainedItem.svelte

```diff
@@ -0,0 +1,20 @@
+<script lang="ts">
+  import { page } from "$app/stores";
+  import { cn } from "$lib/shared/utils";
+  import type { ProcessedSkyBlockItem } from "$lib/types/global";
+  import { Avatar } from "bits-ui";
+  import Image from "lucide-svelte/icons/image";
+
+  export let piece: ProcessedSkyBlockItem;
```

Comment:
If you create new stuff, try using Svelte 5 syntax first so that it's less work when we eventually migrate everything to Svelte 5.

```ts
let { piece }: { piece: ProcessedSkyBlockItem } = $props()
```